### ISSUE TITLE
streamlink 0.2.0 won't launch on Mac OS X 10.9

### ISSUE THREAD

**Issue by fusionfan** [streamlink/streamlink#366]
I have recently upgraded streamlink via pip, but it would give me loads of error messages (see attached file). v 0.1.0 ran fine. Then tried easy_install, and also installation from source. Always the same result. Tried to upgrade all necessary python packages to their latest versions. 
Tried googlink the error messages and various bits of it, either nothing is found, or the solution listed wouldn't work. Apologies I am a bit of a python noob, could anyone help me? 
[streamlink.txt](https://github.com/streamlink/streamlink/files/690154/streamlink.txt)

**Comment by beardypig** [comment]
Try installing the latest version from git
```sh
pip install -U https://github.com/streamlink/streamlink/archive/master.zip
```
That might fix your issue.

**Comment by fusionfan** [comment]
OK, when I do this I get a bunch of "SNIMissingWarning" and "InsecurePlatformWarning". (see attached) However, I have urllib3 version 1.19.1 and python version 2.7.13. None of the solutions lsited here
https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
are valid or working. I start to suspect it might be a python installation issue.
[streamlink2.txt](https://github.com/streamlink/streamlink/files/690189/streamlink2.txt)

**Comment by beardypig** [comment]
`pip` has its own built in `urllib3` you should update `pip` too. 
```sh
pip install -U pip
```
After installing `streamlink` from git what errors do you get now when running `streamlink`?

**Comment by fusionfan** [comment]
So when I run this, I still get the `SNIMissingWarning` and `InsecurePlatformWarning`, see
[streamlink3.txt](https://github.com/streamlink/streamlink/files/690202/streamlink3.txt)
Now when I rerun streamlink the error message is similar: the first few lines about plugins disappeared, but the rest is much the same
[streamlink4.txt](https://github.com/streamlink/streamlink/files/690204/streamlink4.txt)

**Comment by beardypig** [comment]
What does it have at the top of the `/usr/local/bin/streamlink` file? If you run
```sh
head /usr/local/bin/streamlink
```
what does it print? It should have something like 
```
#!/usr/bin/python
```
The run whatever command it lists with `--version` argument, eg. `/usr/bin/python --version` and see what version it reports.

**Comment by fusionfan** [comment]
Oh yay, this should be the issue. I have multiple python installations and they are getting tangled up.
The head of `/usr/local/bin/streamlink` points to `/System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python`
(same for pip etc), which is only 2.7.5. Same for `/usr/bin/python`. Probably this is because I am still using Mavericks (10.9).
However, my default `python` binary is the `macports` version, which is up to date, this is why my version was incorrectly reported. `pip` installed via `macports` is not in the usual `macports` path, that is why the system one is invoked even though `macports` binaries have higher priority in the `$PATH`.

So now I have forced streamlink installation with the `pip` provided by `macports`. This put it under `/opt/local/Library/Frameworks/Python.framework/Versions/2.7/bin/streamlink`.

I have uninstalled the streamlink installation with the system python, and linked the above mentioned streamlink somewhere in the path. And voilá, now it works! Thank you for the help.
